server/src/main/java/org/elasticsearch/action/ingest/SimulatePipelineTransportAction.java

```diff
@@ -52,19 +53,16 @@ public class SimulatePipelineTransportAction extends HandledTransportAction<Simu
     // ThreadLocal because our unit testing framework does not like sharing Randoms across threads
     private final ThreadLocal<Random> random = ThreadLocal.withInitial(Randomness::get);
 
-    @Inject
-    public SimulatePipelineTransportAction(
+    private SimulatePipelineTransportAction(
```

Comment:
Ctor is private and does only initialization stuff (mainly: set private final field)